mitosheet/src/jupyter/api.tsx

```diff
@@ -552,6 +553,20 @@ export default class MitoAPI {
         return undefined;
     }
 
+    async getImportedFilesAndDataframes(): Promise<UpdatedImportObj[] | undefined> {
+        const resultString = await this.send<string>({
+            'event': 'api_call',
+            'type': 'get_imported_files_and_dataframes',
+            'params': {}
+        }, {})
+
+        if (resultString !== undefined && resultString !== '') {
+            return(JSON.parse(resultString).imported_files_and_dataframes);
```

Comment:
We should just be able to return the the results of JSON.parse here - so change what we're returning from the backend so we can just make the types the same on both sides!